TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/TeleOpMovement.kt

```diff
@@ -42,11 +46,18 @@ object TeleOpMovement : Component {
         val joyRadians = atan2(joyY, joyX) - (PI / 3.0)
         var joyMagnitude = sqrt(joyY * joyY + joyX * joyX)
 
+        update()
+
         // if the right bumper is pressed move slower
-        if (gamepad.right_bumper) {
+        if (gamepad.left_bumper) {
             joyMagnitude *= SLOW_MULTIPLIER
         }
 
+        // OutTake drop
+        if (justPressed(GamepadEx.Inputs.RightBumper)) {
```

Comment:
Can you explicitly make `update` and `justPressed` go by `GamepadEx.update` and `GamepadEx.justPressed`? Especially for `update`, its too generic of a name and I think others will be confused.

Reply:
Could you also say that `TeleOpMovement` requires `GamepadEx` to function in the comments?